01_lecture/homework/task2.js

```diff
@@ -0,0 +1,17 @@
+function wrapper(fn) {
+  const bufferOfCached = [];
+  return function () {
+    resultOfOperation = fn.apply(this, arguments);
+    let foundObj = bufferOfCached.find(
```

Comment:
It is better to use `const` type of variable if you are not planning to overwrite it
```suggestion
    const foundObj = bufferOfCached.find(
```

Reply:
done